<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusControl</class>
 <widget class="QWidget" name="StatusControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="statusLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="statusALabel">
      <property name="text">
       <string>General Status A:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="statusBLabel">
      <property name="text">
       <string>General Status B:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusCLabel">
      <property name="text">
       <string>General Status C:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="fdir1StatusLabel">
      <property name="text">
       <string>FDIR Level 1 Status:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="fdir1IMULabel">
      <property name="text">
       <string>FDIR Level 1 IMU Failures:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="fdir1IMUSpinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="fdir2StatusLabel">
      <property name="text">
       <string>FDIR Level 2 Status:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="fdir4StatusLabel">
      <property name="text">
       <string>FDIR Level 4 Status:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="fdir5StatusLabel">
      <property name="text">
       <string>FDIR Level 5 Status:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Extended Status:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTextBrowser" name="extStatusText"/>
    </item>
    <item row="7" column="1">
     <widget class="QTextBrowser" name="fdir5Text"/>
    </item>
    <item row="6" column="1">
     <widget class="QTextBrowser" name="fdir4Text"/>
    </item>
    <item row="5" column="1">
     <widget class="QTextBrowser" name="fdir2Text"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextBrowser" name="fdir1Text"/>
    </item>
    <item row="2" column="1">
     <widget class="QTextBrowser" name="statusCText"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="statusBText"/>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="statusAText"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
